rs-e2e/src/main/resources/assertion_definitions.json

```diff
@@ -111,6 +111,16 @@
         "OBR.count() = 1",
         "OBR-4.1 = '54089-8'"
       ]
+    },
+    {
+      "name": "NTE segment mapping",
+      "conditions": [
+        "MSH-10 = '011'"
+      ],
+      "rules": [
+        "NTE-3 = 'note 1'",
```

Comment:
I understand why we are using this rule but I don't love it since it only works because our HL7 parser only returns the first segment of each type and there is no indexing.

Reply:
Consider using NTE-3 = input.NTE-3 which would work even if we add indexing

Reply:
Great idea

Reply:
Good point. Changed it

Reply:
I reran the automation test as well and it works.